### PR TITLE
Create Design doc and base structure for the Library

### PR DESCRIPTION
**Pull request by Sumsum1231**
Greetings **@kiranharidas187 @5Amogh** ,
I have made the suggested changes and now raising a PR https://github.com/ELEVATE-Project/frontend-authentication-module/pull/10 for the issue https://github.com/ELEVATE-Project/frontend-authentication-module/issues/1 kindly review it.

### CODE REVIEW

**Comment by Sumsum1231** [comment]
Improved some parts, @kiranharidas187 @5Amogh kindly review it.

File: package.json
Comment (5Amogh) [inline package.json:1]: Can you check on why this package.json is empty? and update accordingly

**Review by 5Amogh — CHANGES_REQUESTED** [review]

**Comment by 5Amogh** [comment]
Please raise the PR against release-1.0.0 @Sumsum1231

File: package.json
Comment (5Amogh) [reply package.json:1]: are these two files package.json and package-lock.json are useful anywhere, can you re-check? as they're outside the frontend-auth-module, do they really matter?

**Review by 5Amogh — CHANGES_REQUESTED** [review]

**Pull request merged**